draft-ietf-teep-architecture.md

```diff
@@ -464,9 +464,11 @@ need for TEEP Brokers to share information on installed TAs or resource usage.
 
 ## Multiple TAMs and Relationship to TAs
 
-As shown in {{notionalarch2}}, the TEEP Broker provides connections from the TEE and
-the Untrusted Application to one or more TAMs. The selection of which TAM to communicate with is
-dependent on information from the Untrusted Application and is directly related to the TA.
+As shown in {{notionalarch2}}, a TEEP Broker provides communication between 
+one or more TEEP Agents and
+one or more TAMs. The selection of which TAM to communicate with might be
+made with or without input from an Untrusted Application, but is ultimately
+the decision of a TEEP Agent.
```

Comment:
Considering the case where a TEEP Broker will initiate a call to TAM for some cases, how about the following changes?

"The selection of which TAM to communicate with might be
made with or without input from an Untrusted Application. When a TEEP Broker initiates a request to TAM, it may get such information from an Untrusted Application, the application installer in REE, or other input. When a TEEP Broker initiates a request to TEEP Agent first and then a TAM, the TAM choice may be chosen by the TEEP Agent."

Reply:
Per the transport spec, the TEEP Broker only initiates a call to a TAM at the request of the TEEP Agent.  The TEEP Broker never simply uses the information from the Untrusted Application, it passes it to the TEEP Agent, which then optionally overrides it (or not), before telling the TEEP Broker to connect to the TAM.

As such I do not believe your second sentence is correct.